### PR TITLE
Mantain session

### PR DESCRIPTION
**Pull request by mfmaite**
## Type of change

* [ ] Fix
* [x] Story
* [ ] Chore

## Description of the change

Mantais the user's session and remembers their information

### CODE REVIEW

File: src/networking/controllers/get-user-controller.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+class GetUserController {
+  static async me() {
+    const token = localStorage.getItem('token');
+    return ApiService.get<RawComptUser>(API_ROUTES.ME, {}, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+      },
+    });
+    // return GetUserSerializer.deSerialize(response);
```
Comment (agustinasalmanton) [inline src/networking/controllers/get-user-controller.ts]: comments lines

File: src/pages/unexpected-error/home/home.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+import { ReactComponent as MailSVG } from '../../assets/icons/mail.svg';
+import { ReactComponent as CloseSVG } from '../../assets/icons/close.svg';
+
+const Home = () => {
+  const [openModal, setOpenModal] = useState(false);
+  const closeModal: () => void = () => {
+    setOpenModal(false);
+  };
+  const [inputState, setInputState] = useState('');
+  const [textAreaState, settextAreaState] = useState('random text');
```
Comment (agustinasalmanton) [inline src/pages/unexpected-error/home/home.tsx]: why 'random text' ?

File: src/pages/unexpected-error/home/home.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+import { TextField, TextFieldStatus } from 'common/text-field';
+import { TextArea } from 'common/text-area';
+import { mailFormat } from 'helpers/utils';
+import styles from './home.module.scss';
+import { ReactComponent as MailSVG } from '../../assets/icons/mail.svg';
+import { ReactComponent as CloseSVG } from '../../assets/icons/close.svg';
+
+const Home = () => {
+  const [openModal, setOpenModal] = useState(false);
+  const closeModal: () => void = () => {
```
Comment (agustinasalmanton) [inline src/pages/unexpected-error/home/home.tsx]: What about put this function closeModal() at the bottom of the useState() lines to respect an order in the code?

File: src/pages/unexpected-error/home/home.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    setOpenModal(false);
+  };
+  const [inputState, setInputState] = useState('');
+  const [textAreaState, settextAreaState] = useState('random text');
+  const [helperIState, setHelperIState] = useState('');
+  const [helperTAState, setHelperTAState] = useState('');
```
Comment (agustinasalmanton) [inline src/pages/unexpected-error/home/home.tsx]: why this name? jajajaj

Comment (agustinasalmanton) [reply src/pages/unexpected-error/home/home.tsx]: :stuck_out_tongue_winking_eye::stuck_out_tongue_closed_eyes:

File: src/pages/unexpected-error/home/home.tsx [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+      <h1 className={styles.title}>
+        <Avatar size="xl" />
+        <hr />
+        Welcome! This is the HomePage.
+        <AppLink routeName={RouteName.About}>About</AppLink>
+        <Button onClick={() => setOpenModal(true)}>Hello World</Button>
+      </h1>
+      <div className={styles.formContainer}>
+        <form className={styles.form}>
+          <h3 style={{ backgroundColor: 'lightgray', marginBottom: '48px' }}>Text Field</h3>
```
Comment (agustinasalmanton) [inline src/pages/unexpected-error/home/home.tsx]: It's okey this text?

File: src/pages/unexpected-error/home/home.tsx (hunk 2 above)
Comment (mfmaite) [reply src/pages/unexpected-error/home/home.tsx]: @agustinasalmanton This is from the template, I didn't created it

File: src/pages/unexpected-error/home/home.tsx (hunk 3 above)
Comment (mfmaite) [reply src/pages/unexpected-error/home/home.tsx]: @agustinasalmanton This is from the template, I didn't created it

File: src/pages/unexpected-error/home/home.tsx (hunk 4 above)
Comment (mfmaite) [reply src/pages/unexpected-error/home/home.tsx]: @agustinasalmanton This is from the template, I didn't created it

File: src/pages/unexpected-error/home/home.tsx (hunk 5 above)
Comment (mfmaite) [reply src/pages/unexpected-error/home/home.tsx]: @agustinasalmanton This is from the template, I didn't created it

**Pull request closed without merging**